Event_schema.json

```diff
@@ -285,6 +280,7 @@
                             "type": "number"
                         }
                     ],
+                    "maxItems": 2,
```

Comment:
Apparently we forgot to update the scheme with `generate_json_schema.sh` the last time we changed the `Event_spec.ts`.